### PR TITLE
add Wiwynn's ONIE

### PR DESCRIPTION
**Pull request by JadaLin**
add Wiwynn's ONIE code

### CODE REVIEW

**Comment by cobracmder** [comment]
Hi @JadaLin,

Thanks for sending a PR for this platform.  Would it be possible to squash these commits into one (https://github.com/opencomputeproject/onie/wiki/Contributing#general-patching-philosophy)?

Thanks.

File: machine/wiwynn/andromeda/INSTALL [hunk 1]
```diff
@@ -0,0 +1,116 @@
+Cross-Compiling ONIE
```
Comment (cbrune) [inline machine/wiwynn/andromeda/INSTALL]: The machine directory should be of the form:
```
machine/<vendor>/<vendor>_<machine>
```
For the machine you have defined here, it should look like:
```
machine/wiwynn/wiwynn_andromeda
```

File: machine/wiwynn/andromeda/machine.make [hunk 2]
```diff
@@ -0,0 +1,79 @@
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = BRCM
```
Comment (cbrune) [inline machine/wiwynn/andromeda/machine.make]: For broadcom the SWITCH_ASIC_VENDOR string is "bcm".

File: machine/wiwynn/andromeda/machine.make [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+# The option is significant when I2CTOOLS_ENABLE is 'yes'
+#
+I2CTOOLS_SYSEEPROM = no
+
+# Set the desired kernel version.
+LINUX_VERSION		= 3.14
+LINUX_MINOR_VERSION	= 27
+
+# Set the desired uClibc version
+UCLIBC_VERSION = 0.9.33.2
```
Comment (cbrune) [inline machine/wiwynn/andromeda/machine.make]: you set this variable twice in this file. here and below on line 73.  only do it once, as it is confusing.

File: machine/wiwynn/andromeda/machine.make [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+# Specify any extra parameters that you'd want to pass to the onie linux
+# kernel command line in EXTRA_CMDLINE_LINUX env variable. Eg:
+#
+#EXTRA_CMDLINE_LINUX ?= install_url=http://server/path/to/installer debug earlyprintk=serial
+#
+# NOTE: You can give multiple space separated parameters
+
+# Default to msdos disk label for this platform
+PARTITION_TYPE = msdos
```
Comment (cbrune) [inline machine/wiwynn/andromeda/machine.make]: really?  are you sure you want msdos?  gpt is much preferred.

**Comment by JadaLin** [comment]
thanks, i update the config and rename folder name as comment, will commit again

**Comment by JadaLin** [comment]
To master, i renew my commit here, 
https://github.com/opencomputeproject/onie/pull/376/commits/eae861585b2e8815032c66c80c45740a3d6c17a5,
plz review and advise.

**Pull request closed without merging**